.github/workflows/main.yml

```
@@ -13,72 +13,72 @@ jobs:
+          key: dummy-app-node-modules-cache-${{ hashFiles('spec/dummy/package.json') }}-${{ matrix.versions }}
+      - name: yalc add react-on-rails
+        run: cd spec/dummy && yalc add react-on-rails
+      - name: Install Node modules with Yarn for dummy app
+        run: cd spec/dummy && yarn install --no-progress --no-emoji
+      - name: Save dummy app ruby gems to cache
+        uses: actions/cache@v4
+        with:
+          path: spec/dummy/vendor/bundle
+          key: dummy-app-gem-cache-${{ hashFiles('react_on_rails.gemspec') }}-${{ hashFiles('Gemfile.development_dependencies') }}-${{ matrix.versions }}
+      - name: Install Ruby Gems for dummy app
+        run: cd spec/dummy && bundle lock --add-platform 'x86_64-linux' && bundle check --path=vendor/bundle || bundle _2.5.9_ install --path=vendor/bundle --jobs=4 --retry=3
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2015:info:1:60: Note that A && B || C is not if-then-else. C may run when A is true

(shellcheck)